StackExchange.Redis/StackExchange/Redis/GeoEntry.cs

```diff
@@ -0,0 +1,144 @@
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace StackExchange.Redis
+{
+
+    /// <summary>
+    /// Describes a GeoEntry element with the corresponding value
+    /// GeoEntries are stored in redis as SortedSetEntries
+    /// </summary>
+    public struct GeoEntry : IEquatable<GeoEntry>, IEquatable<SortedSetEntry>, IComparable, IComparable<GeoEntry>, IComparable<SortedSetEntry>
```

Comment:
The whole Geo concept in redis is really just a Sorted Set with some new methods that operate on them. My thought was it should be presented this way. My initial thought was the GeoEntry should inherit from SortedSetEntry, but it was represented as a struct , and I didnt want to change that w/o your input.